packages/tsup-configs/src/defineConfig.ts

```diff
@@ -0,0 +1,32 @@
+import { defineConfig as _defineConfig, type Options } from "tsup";
+
+export type DefineConfigOptions = Parameters<typeof _defineConfig>[0];
+
+export function defineConfig(baseOptions: Options, userOptions?: DefineConfigOptions | undefined): ReturnType<typeof _defineConfig> {
```

Comment:
`| undefined  ` is repetitive, you already put this parameters optional with the ?

Reply:
Yeah I am not sure about that `| undefined`, this is something you added to the original function.

Was it a typo and it's necessary?

Reply:
in the original code `config: DefineConfigOptions | undefined`  is ok. config is not followed by a ?.

In this case userOptions?: DefineConfigOptions | undefined, it's repetitive to have a ? and a | undefined

```
export function defineConfig(baseOptions: Options, userOptions?: DefineConfigOptions): ReturnType<typeof _defineConfig> {
```